app/views/jobs/_form.html.erb

```diff
@@ -1,97 +1,162 @@
-              Remote work
-            </label><i class="fa fa-wifi"></i>
+<%= form_with model: @job ,html: { multipart: true } do |f| %>
+<section class="vh-800" style="background-color: #2779E2;">
+  <div class="container h-1000">
+    <div class="row d-flex justify-content-center align-items-center h-1000">
+      <div class="col-xl-10">
+      <br>
+        <h1 class="text-white mb-4">Tell us about the position</h1>
+        <div class="card" style="border-radius: 15px;">
+          <div class="card-body">
+            <div class="row align-items-center pt-4 pb-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Headline</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.text_field :title,  :class =>"form-control form-control-lg" , :placeholder=>"One line about Job Role"   %>
+              </div>
+            </div>
+            <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Job Type</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.select :job_type,["Select the job type","Full-time employment", "Part-time", "short-timeContract", "Freelance", "Internship","Volunteer contribution","Partner for a venture"],{},{:class=>"form-control form-control-lg"}%>
+              </div>
+            </div>
+             <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Job Category</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.select :category,["Select the job category","Programming","Interaction Design", "Graphic Design","Information Security",
+                 "Testing/Quality Assurance","IT/System Administration","Business/Management",
+                "Writer/Editor","Customer Support","Office Administation"],{},{:class=>"form-control form-control-lg"}%>
+              </div>
+            </div>
+            <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Location</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.select :location,["Select the location","Ariyalur",    "Chennai",    "Coimbatore",    "Cuddalore",    "Dharmapuri",    "Dindigul",    "Erode",    "Kanchipuram",    "Kanyakumari",    "Karur",    "Krishnagiri",    "Madurai",    "Nagapattinam",    "Namakkal",    "Nilgiris",    "Perambalur",    "Pudukkottai",    "Ramanathapuram",    "Salem",    "Sivaganga",    "Thanjavur",    "Theni",    "Thoothukudi",    "Tiruchirappalli",    "Tirunelveli",    "Tiruppur",    "Tiruvallur",    "Tiruvannamalai",    "Tiruvarur",    "Vellore",    "Viluppuram",    "Virudhunagar"],{},{:class=>"form-control form-control-lg"}%>
+              <br>
```

Comment:
don't hardcode data in views, this should go in controller or model but not in the views